src/openrct2/management/Research.h

```diff
@@ -11,6 +11,7 @@
 
 #include "../common.h"
 #include "../object/ObjectLimits.h"
+#include "../rct2/RCT2.h"
```

Comment:
Stray change

Reply:
It's actually required. But really should not be. The research should do the conversion in the rct2 files.